firewall.tf

```diff
@@ -0,0 +1,35 @@
+# Firewall rules.
+
+resource "google_compute_firewall" "serverpod-instance" {
+  name = "serverpod-${var.runmode}-instance"
+
+  network       = google_compute_network.serverpod.name
+  source_ranges = ["0.0.0.0/0"]
+
+  # allow {
```

Comment:
I find this in several examples, but is it needed for anything? Everything seems to work without it.